include/cpr/ssl_options.h

```diff
@@ -14,6 +20,324 @@ class VerifySsl {
     bool verify_ = true;
 };
 
+namespace ssl {
+
+// set SSL client certificate
+class CertFile {
+  public:
+    template <typename FileType>
+    CertFile(FileType&& p_filename) : filename(CPR_FWD(p_filename)) {}
+
+    std::string filename;
+
+    virtual const char* GetCertType(void) const {
+        return "PEM";
+    }
+};
+
+typedef CertFile PemCert;
```

Comment:
Is this `typedef` really necessary?

Reply:
It's just an alias for `PemCert` because `CertFile` may confuse the user who need use a `PEM` cert file.

Reply:
Why then don't call it `PemCert` in the first place?

Reply:
Because the `CertFile` will pass to `SetOption` as argument, it will be a little strange to pass a `DerCert` to a argument which type is `PemCert`.

```cpp
    void SetOption(const ssl::CertFile& opt) {
        cert_file = opt.filename;
        cert_type = opt.GetCertType();
    }
    void SetOption(const ssl::KeyFile& opt) {
        key_file = opt.filename;
        key_type = opt.GetKeyType();
        key_pass = opt.password;
    }
```

Reply:
OK.